_layouts/eip.html

```diff
@@ -92,3 +92,11 @@ <h2>Citation</h2>
     "copyrightYear": "{{ page.created | date: "%Y" }}"
   }
 </script>
+<script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.3/MathJax.js?config=TeX-MML-AM_CHTML">
```

Comment:
Nice! But I wonder if there is any security concern? e.g. consider add MD5 checksum for loading MathJax

Reply:
Good catch!